### DISCUSSION TITLE
why grpc.max_send_msg_size option out of work without TLS

### DISCUSSION THREAD

**Discussion by lyt8384** [roadrunner-server/roadrunner#1384] [Q&A] [answered]
https://github.com/roadrunner-server/grpc/blob/5cb28e5e133124f82daa9e3905f3f4b54c9f07db/server.go#L116
![image](https://user-images.githubusercontent.com/3069157/204953055-52f9ff92-9392-4f60-9ed3-d0a9fee91293.png)

Looks like the tls must be configured, if i want to configure the grpc.max_send_msg_size option? why?

**Comment by rustatian** [comment, marked as answer]
Hey @lyt8384 👋🏻 

Yeah, looks like a configuration misplace; I'll fix that; thanks for pointing it out 👍🏻

**Reply by rustatian** [reply]
Done, it will be in the 2.12.1 today. Thanks ⚡

**Reply by lyt8384** [reply]
thank you
